packages/readline7.rb

```diff
@@ -1,37 +1,30 @@
 require 'package'
 
 class Readline7 < Package
+
+  @patchlevel = 5
+
   description 'The GNU Readline library provides a set of functions for use by applications that allow users to edit command lines as they are typed in.'
   homepage 'http://cnswww.cns.cwru.edu/php/chet/readline/rltop.html'
-  version '7.0p3'
-  source_url 'ftp://ftp.gnu.org/gnu/readline/readline-7.0.tar.gz'
+  version "7.0p#{@patchlevel}"
+  source_url 'https://ftp.gnu.org/gnu/readline/readline-7.0.tar.gz'
```

Comment:
Can you please change this line to `source_url 'https://ftpmirror.gnu.org/readline/readline-7.0.tar.gz'`?